src/ctl.c

```diff
@@ -3457,3 +3467,32 @@ experimental_arenas_i_pactivep_ctl(tsd_t *tsd, const size_t *mib,
 	malloc_mutex_unlock(tsd_tsdn(tsd), &ctl_mtx);
 	return ret;
 }
+
+static int
+experimental_prof_recent_alloc_max_ctl(tsd_t *tsd, const size_t *mib,
+    size_t miblen, void *oldp, size_t *oldlenp, void *newp, size_t newlen) {
+	int ret;
+
+	if (!(config_prof && opt_prof && prof_active)) {
```

Comment:
Should we leave prof_active out of this? I can imagine we want to set the max count first, and then activate prof.

Reply:
Makes sense. Also adjusted associated tests.